File: exocps.java
import java.io.*;
import java.util.*;

public class Main {
    private static final String FILE_PATH = "payments.txt";

    private static void printMenu() {
        System.out.println("""
            *********************************
            *  Online College Payment System *
            *********************************
            1. Process Payment
            2. Delete Payment
            3. Edit Payment
            4. Search Payment
            5. Print All Payments
            6. Exit
            """);
    }

    private static void makePayment(Scanner sc) throws IOException {
        System.out.print("Enter Student ID: ");
        String id = sc.nextLine();
        System.out.print("Enter Amount: RM");
        String amount = sc.nextLine();
        
        try (FileWriter fw = new FileWriter(FILE_PATH, true)) {
            fw.write(id + "," + amount + "\n");
            System.out.println("Payment saved!");
        }
    }

    private static void deletePayment(Scanner sc) throws IOException {
        System.out.print("Enter Student ID to delete: ");
        String targetId = sc.nextLine();
        
        List<String> lines = new ArrayList<>();
        try (BufferedReader br = new BufferedReader(new FileReader(FILE_PATH))) {
            String line;
            while ((line = br.readLine()) != null) {
                if (!line.startsWith(targetId + ",")) {
                    lines.add(line);
                }
            }
        }
        
        try (FileWriter fw = new FileWriter(FILE_PATH)) {
            for (String line : lines) {
                fw.write(line + "\n");
            }
        }
        System.out.println("Payment deleted!");
    }

    public static void main(String[] args) throws IOException {
        new File(FILE_PATH).createNewFile(); // Create if not exists
        Scanner sc = new Scanner(System.in);
        
        while (true) {
            printMenu();
            String choice = sc.nextLine();
            switch (choice) {
                case "1" -> makePayment(sc);
                case "2" -> deletePayment(sc);
                case "6" -> System.exit(0);
                default -> System.out.println("Invalid choice!");
            }
        }
    }
}

File: exocps.rs
use std::fs::{File, OpenOptions};
use std::io::{self, BufRead, BufReader, Write};
use std::path::Path;

const FILE_PATH: &str = "payments.txt";

fn print_menu() {
    println!(
        "
*********************************
*  Online College Payment System *
*********************************
1. Process Payment
2. Delete Payment
3. Edit Payment
4. Search Payment
5. Print All Payments
6. Exit"
    );
}

fn make_payment() -> io::Result<()> {
    println!("Enter Student ID:");
    let mut id = String::new();
    io::stdin().read_line(&mut id)?;

    println!("Enter Amount (RM):");
    let mut amount = String::new();
    io::stdin().read_line(&mut amount)?;

    let mut file = OpenOptions::new()
        .append(true)
        .create(true)
        .open(FILE_PATH)?;
    writeln!(file, "{},{}", id.trim(), amount.trim())?;
    println!("Payment saved!");
    Ok(())
}

fn delete_payment() -> io::Result<()> {
    println!("Enter Student ID to delete:");
    let mut target_id = String::new();
    io::stdin().read_line(&mut target_id)?;
    let target_id = target_id.trim();

    let file = File::open(FILE_PATH)?;
    let reader = BufReader::new(file);

    let mut lines = Vec::new();
    let mut found = false;

    for line in reader.lines() {
        let line = line?;
        if !line.starts_with(&format!("{},", target_id)) {
            lines.push(line);
        } else {
            found = true;
        }
    }

    if !found {
        println!("Payment not found!");
        return Ok(());
    }

    let mut file = File::create(FILE_PATH)?;
    for line in lines {
        writeln!(file, "{}", line)?;
    }
    println!("Payment deleted!");
    Ok(())
}

fn main() -> io::Result<()> {
    if !Path::new(FILE_PATH).exists() {
        File::create(FILE_PATH)?;
    }

    loop {
        print_menu();
        let mut choice = String::new();
        io::stdin().read_line(&mut choice)?;
        match choice.trim() {
            "1" => make_payment()?,
            "2" => delete_payment()?,
            "6" => std::process::exit(0),
            _ => println!("Invalid choice!"),
        }
    }
}